checkov/terraform/module_loading/loaders/local_path_loader.py

```diff
@@ -10,8 +10,12 @@ def __init__(self) -> None:
         self.is_external = False
 
     def _is_matching_loader(self) -> bool:
-        return self.module_source.startswith("./") or self.module_source.startswith("../") \
-               or self.module_source.startswith(self.current_dir) or self.module_source.startswith('/')
+        return (
+            self.module_source.startswith("./")
+            or self.module_source.startswith("../")
+            or self.module_source.startswith(self.current_dir)
+            or self.module_source.startswith("/")
```

Comment:
@gruebel Merge from master, as this line was removed. Abs paths will be wrong in different machines so we don't try to initiallize them

Reply:
actually the absolute path is needed, otherwise it won't work, because in the `parser.py` the module source is changed to an abs path.
https://github.com/bridgecrewio/checkov/blob/0f63f8e5be52d2a4b9c1b46d7952eddb717e673c/checkov/terraform/parser.py#L389-L391